### PR TITLE
Adding Package Dependencies

### PR DESCRIPTION
**Pull request by arshadparwaiz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,31 +4,35 @@
...
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^27.2.3",
+    "eslint-plugin-jest": "^26.5.3",
     "eslint-plugin-jsdoc": "^39.3.3",
     "eslint-plugin-n": "^15.2.3",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
-    "jest": "^29"
+    "jest": "^27.2.4"
```
Comment (sukamat) [inline package.json:31]: Why downgrade? Here and few other places..

File: package.json [hunk 2]
```diff
@@ -4,31 +4,35 @@
   "private": true,
   "dependencies": {
     "@adobe/aio-lib-core-logging": "^2.0.0",
+    "@adobe/aio-lib-files": "^3.0.1",
+    "@adobe/aio-lib-state": "^2.1.2",
```
Comment (sukamat) [inline package.json:8]: Are we using lib-state? It will be good to add only those libs that will be used and update later as necessary.

**Review by sukamat — COMMENTED** [review]
@arshadparwaiz It will be good to update the package.json file as and when new features get added and to update it as part of the same PR where the feature gets added. It will be easier to keep this file clean than starting with a uber set of libs that may not be used. It will also avoid any potential conflicts that might arise.

File: package.json (hunk 1 above)
Comment (arshadparwaiz) [reply package.json:31]: will update this.. thanks

**Comment by arshadparwaiz** [comment]
> @arshadparwaiz It will be good to update the package.json file as and when new features get added and to update it as part of the same PR where the feature gets added. It will be easier to keep this file clean than starting with a uber set of libs that may not be used. It will also avoid any potential conflicts that might arise.

Sure Sunil. As of now the build is expecting few dependencies, will keep it to just those, and add later as needed

**Comment by arshadparwaiz** [comment]
After updating the local env, we are no more getting any dependency errors. Closing the PR.

**Pull request closed without merging**